### ISSUE TITLE
bug(Youtube): Watch this content on the latest version of YouTube instead

### ISSUE THREAD

**Issue by Domiiniik** [ReVanced/revanced-patches#2878]
### Bug description

https://github.com/ReVanced/revanced-patches/assets/30195961/d0ae72f2-49db-4174-9962-fd9fd39c19ae

Possibly fixed with

- https://github.com/microg/GmsCore/issues/2253

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by Domiiniik** [comment]
It only happens with 1 particular video for some strange reason?

**Comment by Domiiniik** [comment]
It now happens with all videos, wtf

**Comment by oSumAtrIX** [comment]
Send a recording of the following steps:

![image](https://github.com/ReVanced/revanced-patches/assets/13122796/4662e42b-0d65-4db8-89d9-7e50c4e2f04e)

**Comment by Domiiniik** [comment]
I'm already using latest version, with latest patches from latest manager tho it worked fine until now, there is no need to record the repatching process all over again

**Comment by oSumAtrIX** [comment]
Please attach the requested video. If it turns out to be a problem on our end I'll reopen the issue.

**Comment by Domiiniik** [comment]
There "Watch this content on the latest version of Youtube' is displayed

https://github.com/ReVanced/revanced-patches/assets/30195961/d0ae72f2-49db-4174-9962-fd9fd39c19ae

**Comment by oSumAtrIX** [comment]
What about the unpatched app?

**Comment by Domiiniik** [comment]
Original Youtube works fine

**Comment by oSumAtrIX** [comment]
Try in incognito or a different account

**Comment by Domiiniik** [comment]
Turning off spoofing signature makes it work and not display this message but it stops after a minute:
![Screenshot_20240314_220631_YouTube](https://github.com/ReVanced/revanced-patches/assets/30195961/9e27d49e-76a1-4446-bb64-57c3257c6a3b)
The issue seems to be with signature spoof

**Comment by Domiiniik** [comment]
Don't have a different network, I'm on mobile data

**Comment by Domiiniik** [comment]
Total observations:
A) Videos play normally in Incognito Mode
B) Videos display "Watch this content in the latest version of Youtube" if spoofing signature is on
B) Videos play normally if spoofing signature is off (only for about a minute obviously, like we know)
C) I don't have a different account to test, this is my only main account that I'm using for about 8 years. And that I've used Vanced too in the past and now Revanced.
D) I Don't have a different network to test on, running only mobile data..
E) it all worked fine since the latest patches were released, this issue only occurred today and now.

**Comment by oSumAtrIX** [comment]
To investigate, I'll need access to the account. Please reach out to me at osumatrix@revanced.app.

**Comment by Domiiniik** [comment]
Email sent. Waiting on action

**Comment by Malex21** [comment]
I have the exact same issue as @Domiiniik here, and I also have videos stopping after around a minute when not using Spoof App Version !

**Comment by oSumAtrIX** [comment]
Commit 44a8a1399897caaff3ff45db8549ddedb2f01b68 _may_ have fixed the issue. I can not test it because I can not reproduce the problem. Someone affected will have to test.

**Comment by bwskin** [comment]
If I did everything correctly installing pre-release patches (last time used CLI long time ago), issue still persists.

**Comment by Domiiniik** [comment]
Can anybody get me a test app? I don't have a pc to use cli.

**Comment by Domiiniik** [comment]
@LisoUseInAIKyrios can you point me to a guide how to build dev branches with termux?

**Comment by stu2005** [comment]
I'm having the same issue with my account.  I can provide information if needed.

**Comment by oSumAtrIX** [comment]
I am not exactly sure but https://github.com/microg/GmsCore/issues/2253 looked like something that could be the cause of playback issues
